pkg/diagnostics/utils/trace_utils.go

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package utils
+
+import (
+	"strconv"
+)
+
+const (
+	defaultTraceSamplingRate = 0.001
```

Comment:
Can we use 1e-4 to follow opencensus default rate - https://github.com/census-instrumentation/opencensus-go/blob/46dfec7deb6e8c5d4a46f355c0da7c6d6dc59ba4/trace/sampling.go#L21 ?

Reply:
done.